docker-compose.yml

```diff
@@ -131,7 +131,6 @@ services:
       args:
         LLVM_VERSION: 7
     environment:
-      ARROW_FLIGHT: "ON"
```

Comment:
Why? (no objection, just curious the connection)

Reply:
because most Flight dependencies can't use the system packages and needs to download protobuf, grpc, liburiparser. This reduce in half the time to compile. I think flight should be it's own CI entry (or a nightly job).